site-src/concepts/versioning.md

```diff
@@ -1,116 +1,70 @@
 # Versioning
 
-## Summary
-Each Gateway API release is represented by a bundle version that represents
-that specific combination of CRDs, API versions, and validating webhook. To
-enable experimental fields, future releases of the API will include stable and
-experimental CRD tracks. Users will be able to access experimental features by
-installing the experimental CRDs. A cluster can contain either an experimental
-or stable CRD for any resource at a given time. 
+## Overview
+Each new release of Gateway API is defined with a "bundle version" such as
+v0.4.0. This contains the following:
 
-## Version Terminology
-Gateway API has two related but different versioning schemes:
-
-### 1. API Versions (ex: v1alpha2)
-Each API version provides a unique way to interact with the API. Significant
-changes such as removing or renaming fields will be represented here.
-
-### 2. Bundle Versions (ex: v0.4.0)
-Bundle versions are broader in scope. They use semantic versioning and include
-the following:
-
-* API Types/CRDs
+* API Types
+* CRDs
 * Validating Webhook
```

Comment:
```suggestion
* Validating Webhook install
```

Reply:
I'm not sure I understand this change. I think the bundle version includes a webhook and CRDs, which you can separately install in a cluster.

Reply:
Yeah, agreed, this is more confusing. Resolving.